scholarx/2022/index.html

```diff
@@ -121,12 +121,12 @@ <h1 class="display-3 text-center mb-5">Program Timeline</h1>
 
 <!-- success stories section -->
 <section class="section"
-    style="display: flex; align-items: center; justify-content: center; padding-bottom: 130px;">
+    style="display: flex; align-items: center; justify-content: center; background-color: #f4f5f7;">
```

Comment:
You can use the `bg-secondary` class instead of setting the background colour manually.

Reply:
oh. Okay Thank you